### ISSUE TITLE
Communicating content types to users

### ISSUE THREAD

**Issue by DBishton** [OpenDevelopmentMekong/wp-odm_theme#155]
Taking a learnt lesson from our last interface redesign, I suggest we begin to prepare some resources to help users adjust. 

I think the simplicity of the proposed design and *genuinely* integrated interface is a great improvement over the last and users shouldn't need to have their hands held too much on how to access anything. However I feel that we should spell out the various content types -- it's probably not obvious to a new user that a 'topic' has a precise definition, or how it differs from a 'story'. I'm raising this as an issue here to discuss an approach to communicating this, and how that fits into the design.

My suggestion is a persistent approach -- help information should be able to be updated on the site as new features are added or existing ones evolve, as opposed to a 'dated' approach such as a YouTube walkthrough that cannot be easily updated.  

To introduce content types, I suggest a small 'i' or '?' icon that could be added next to the title of each content type on the home page, presenting the user with

a) a popup describing that content type, or
b) a page describing all content types, with a popup informing the user that clicking will take them to our FAQ/Help/Sitemap.

@acorbi @gimmemochi @jpeizer1 @prustar thoughts on this approach? 

![image](https://cloud.githubusercontent.com/assets/1206387/16939588/369c449c-4dad-11e6-9702-c74d13f92564.png)

**Comment by jpeizer1** [comment]
@acorbi @gimmemochi @prustar @dbishton   Agreed -- -- It would be simpler if they could be taken to a page that introduced all the different types although if we wanted to offer a variety of different helps (so that it would get longer and longer) then a bunch of separate windows specific to each help issue would be the better  solution.

**Comment by acorbi** [comment]
@DBishton @jpeizer1 @gimmemochi @prustar I like the idea as well. I would suggest to have a dedicated "getting started" page (as in wordpress page, linked from the footer menu if wished, see https://pp.opendevelopmentmekong.net/getting-started/) where all those infos are maintained. Then we would link those question marks to the corresponding sections of the page. That way, we manage this issue with editorial processes and do not rely on code.
